src/handleCommand.ts

```diff
@@ -73,7 +71,7 @@ export class CommandHandler implements Handler {
     }
     return new Promise(r => {
       // TODO: remove shell: true once redirections and other stuff like that are finished
-      const child = spawn(commandName, args, { env: { ...variables, ...process.env }, cwd, shell: true });
+      const child = spawn(commandName, args, { shell: true });
```

Comment:
https://github.com/nearlySplat/splatsh/pull/15/commits/bbe6d6040f0a883a48a89506b00515046b47de17#r632915255

Reply:
idk what that's supposed to mean

passing cwd is no longer necessary as process.cwd() is used by default and the session variables should not be used as environment  variables, they're simply temporary variables in the current shell so I removed the env part too